manifest.yml

```diff
@@ -5,4 +5,4 @@ applications:
   buildpacks:
   - https://github.com/cloudfoundry/go-buildpack
   env:
-    GOVERSION: go1.20
+    GOVERSION: go1.21
```

Comment:
Looks like go1.22 is supported: https://github.com/cloudfoundry/go-buildpack/releases

Can we go to that?